4-HTTP/client/client.js

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+
+const baseURL = "http://localhost:3000";
+const instance = axios;
+
+const getToDoList = async function () {
```

Comment:
you don't have to define this function as `async`, it will return a promise anyway